unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/DecentralandEntity.cs

```diff
@@ -32,8 +32,10 @@ public class DecentralandEntity : DCL.ICleanable, DCL.ICleanableEventDispatcher
         public System.Action<DCLTransform.Model> OnTransformChange;
         public System.Action<DecentralandEntity> OnMeshesInfoUpdated;
         public System.Action<DecentralandEntity> OnMeshesInfoCleaned;
+        public event Action<bool> OnEntityBoundsCheckerStatusChanged;
```

Comment:
Can't we have this event in `SceneBoundsChecker` itself? Having events for externals systems like this is kind of an anti-encapsulation pattern that grew over time.

I figure we can add some `Action<DecentralandEntity, bool>` that should do the trick? We want to listen for all the entities state change anyways.

Reply:
You're right, after our talk from yesterday I think it has more sense to move this responsability to the `SceneBoundsChecker` and not to overload more the `DecentralandEntity` class. You can see the change in this commit: https://github.com/decentraland/explorer/pull/2217/commits/30a9e6826f87eebdf89236b6696d7e48025d03cb Please take a quick look, thanks for the feedback, @BrianAmadori!